css/style.css

```diff
@@ -0,0 +1,2912 @@
+    margin-left: 5px;
+  }
+
+  .button__red {
+    padding-left: 18px;
+    padding-right: 18px;
+    padding-bottom: 7px;
+    padding-top: 7px;
+    margin-right: 5px;
+    margin-left: 0;
+    margin-top: 0;
+  }
+
+  .button__text {
+    margin-right: 0;
+    padding-right: 0;
+    padding-left: 0;
+    font-size: 13px;
+  }
+
+  .bottom__description-title {
+    width: 320px;
+  }
+    
+  .middle__inner {
+    background: url(../img/tesla-mobil2.png) no-repeat;
+  }
+
+  .middle {
+    max-width: 375px;
+    margin-top: 0;
+  }
+
+  .middle__title {
+    display: none;
+  }
+    
+  .middle__info {
+    max-width: 375px;
+    margin-top: 320px;
+    height: 300px;
+  }
+
+  .middle__info-row {
+    max-width: 400px;
+    height: 304px;
+  }
+
+  .middle__content {
+    max-width: 375px;
+  }
+
+  .middle__info-title {
+    margin-top: 17px;
+    margin-left: 26px;
+    margin-bottom: 10px;
+    font-size: 30px;
+    line-height: 37px;
+  }
+
+  .middle__info-text {
+    max-width: 300px;
+    margin-top: 0px;
+    margin-left: 27px;
+    font-size: 15px;
+    line-height: 18px;
+  }
+
+  .middle__row {
+    display: none;
+  }
+
+  .button__btn {
+    display: none;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-size: 400px 80px;
+    background-position: 0 50px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-mobile.png) no-repeat;
+    background-position: 0 60px;
+  }
+
+  .bottom__info {
+    margin-top: 297px;
+    width: 400px;
+    height: 300px;
+  }
+    
+  .bottom__description {
+    padding-top: 16px;
+    padding-left: 31px;
+    padding-right: 10px;
+  }
+
+  .bottom__description-title {
+    width: 320px;
+    margin-top: 2px;
+  }
+
+  .bottom__description-text {
+    padding-top: 12px;
+  }
+
+  .bottom__stars {
+    display: none;
+  }
+} 
+
+@media (max-width: 373px) {
+  .top {
+    background: url(../img/tesla-fon.png) ;
+    background-position: -18px 80px;
+    background-repeat: no-repeat;
+    background-size: 370px 120px;
+  }
+
+  .top__wrapper-charset {
+    margin-left: 10px;
+    background: url(../img/svg/road-mobile.svg) no-repeat;
+    background-position: 150px 2px;
+  }
+  .charset {
+    width: 120px;
+  }
+
+  .charset__speed {
+    margin-left: 0;
+    margin-right: 5px;
+  }
+
+  .charset__item {
+    width: 100px;
+  }
+
+  .charset__first-small {
+    margin-right: 0;
+    padding-right: 0;
+  }
+
+  .top__wrapper-item {
+    margin-right: 0;
+  }
+
+  .button__outline {
+    padding-left: 10px;
+    padding-right: 0;
+    margin-left: 0;
+  }
+
+  .button__red {
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .middle__content {
+    width: 320px;
+  }
+
+  .middle__info-title {
+    margin-left: 3px;
+  }
+
+  .middle__info-text {
+    margin-left: 5px;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-size: 300px 80px;
+    background-position: 0 50px;
+}
+
+.bottom__inner {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)